h/accounts/test/views_test.py

```diff
@@ -775,111 +801,102 @@ def test_get_when_logged_in_already_logged_in_to_different_account(self):
             "You're already signed in to a different account")
 
 
-@pytest.mark.usefixtures('routes_mapper')
 class TestProfileController(object):
 
-    def test_post_400s_with_no_formid(self):
-        user = FakeUser()
-        request = DummyRequest(post={}, authenticated_user=user)
+    @pytest.fixture(autouse=True)
```

Comment:
We have to call `config.begin(request)` for these tests, unfortunately, so that `pyramid.threadlocal.get_current_request` returns the correct request. This is because `deform_jinja2` calls `get_current_request`...